src/components/displaySvg/finalSvgDisplay.scss

```diff
@@ -0,0 +1,67 @@
+.display-page-div{
+    padding: 171px;
+    width: 373px; 
+}
+
+ellipse:hover,rect:hover,path:hover,text:hover,linearGradient:hover,polygon:hover,circle:hover {
+   background-color: yellow;
+   fill: yellow;
+}
+
+ .inline-display-position{
+   padding-top: 71%;
+ }
+ svg{
+    max-height: 106px;
+    max-width: fit-content;
+   //  background-color: white;
+ }
+  
+ .display-svgs-border-match {
+    max-height: 381px;
+    border: outset;
+    border-color: green;
+}
+
+ .display-original-optimised{
+    overflow-y: scroll;
+    max-height: 571px;
+    border: outset;
+    border-color: green;
+ }
+
+ .button-background{
+    background-color: grey;
+    height: 171px;   
+ }
+ .position-loader{
+   padding-top: 11%;
+}
+
+ .loader {
+    border: 16px solid greenyellow;
+    border-radius: 56%;
```

Comment:
try using px here

Reply:
Its starts to be perfect round at     border-radius: 120 px; , I did changed it.

Reply:
cool